packages/atomic/src/components/atomic-no-results/atomic-no-results.tsx

```diff
@@ -27,6 +27,7 @@ import {AriaLiveRegion} from '../../utils/accessibility-utils';
  * @part no-results - The text indicating that no results were found for the search.
  * @part search-tips - The search tips to help the user correct the query.
  * @part highlight - The highlighted query.
+ * @part icon - The magnifying glass icon.
```

Comment:
Could it be ONLY a magnifying glass..? I mean, the name of the part is very generic, just `icon`.

Reply:
NVM, I see the answer below 😅 Yep, it could be only a magnifying glass.